terraform/modules/google-cloud/apps/relay/main.tf

```diff
@@ -404,37 +404,6 @@ resource "google_compute_firewall" "http-health-checks" {
   }
 }
 
-# Allow inbound traffic
```

Comment:
Just want to confirm, the only inbound traffic Relays will ever need will be between this range: `49152 - 65535`?

Reply:
And 3478 (stun), 80 (healthcheck), 22 (ssh)